### PR TITLE
LPS-73738 Align the status label to the horizontal center of the info bar

### PR DESCRIPTION
**Pull request by jeyvison**

### CODE REVIEW

**Comment by pedroqueiroz94** [comment]
@clovisdasilvaneto , @alinedoleron can you guys take a look at the CSS here?

File: modules/apps/forms-and-workflow/portal-workflow/portal-workflow-web/src/main/resources/META-INF/resources/css/_definition.scss [hunk 1]
```diff
@@ -11,6 +11,7 @@
 }
 
 .info-bar-item {
+	margin-top: -3px;
```
Comment (clovisdasilvaneto) [inline modules/apps/forms-and-workflow/portal-workflow/portal-workflow-web/src/main/resources/META-INF/resources/css/_definition.scss]: Hey @jeyvison, why did you put a negative margin in the .info-bar-item component? Let me know your thinking. Depending of the context, you can put a negative top property, like: `top: -3px`, but it's not a good practice to use negative measures of margins or positions.

Comment (jeyvison) [reply modules/apps/forms-and-workflow/portal-workflow/portal-workflow-web/src/main/resources/META-INF/resources/css/_definition.scss]: Hey @clovisdasilvaneto , I talked with Matusalem about it when we decided to go this way but i have no problem on changing it if the result is the same.

**Comment by pedroqueiroz94** [comment]
LGTM

**Comment by rafaprax** [comment]
Just started reviewing :) <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

**Comment by rafaprax** [comment]
Pull request submitted to https://github.com/brianchandotcom/liferay-portal/pull/55472. [See changes here.](https://github.com/rafaprax/liferay-portal/compare/0e59a4c07e5f~1...0e59a4c07e5f) <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

**Pull request closed without merging**